lib/testing-components/Test.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect } from "react";
+import {
+  storyblokInit,
+  apiPlugin,
+  getStoryblokApi,
+  useStoryblokBridge,
+  StoryblokComponent,
+} from "gatsby-source-storyblok";
+
+const Test = ({ bridge, accessToken, components, blok }) => {
+  storyblokInit({
+    accessToken,
+    bridge,
+    use: accessToken ? [apiPlugin] : [],
+    components,
+  });
+
+  const storyblokApi = getStoryblokApi();
+  const apiExists = !!(storyblokApi && typeof storyblokApi.get === "function");
+
+  useEffect(() => {
+    useStoryblokBridge(43423, (newStory) => console.log(newStory));
```

Comment:
Should we test `useStoryblokState` here?
Is this `useStoryblokBridge` function different from the one from `@storyblok/react`?

Reply:
I think it could be considered the same except that you just need to be careful about what to store as an id in the first param.

(Gatsby's `id` is String but `internalId` from Gatsby retrieves the id format we need (Number).)
If it's the same as React SDK, we don't need it, don't we? (But Gatsby SDK uses an alias. Do we need it in this case?)🤔

Reply:
Yeah... Gatsby do some weird things so Gatsby SDK's `useStoryblokState` differs a bit from the react one

Reply:
🚀